command/root.go

```diff
@@ -250,6 +250,30 @@ func determineBaseRepo(cmd *cobra.Command, ctx context.Context) (ghrepo.Interfac
 	return baseRepo, nil
 }
 
+func convertRepoInterfaceToRepository(ctx context.Context, repo ghrepo.Interface) (*api.Repository, error) {
```

Comment:
I've added this function so we can get an *api.Repository object from a ghrepo.Interface.

Reply:
Is it worth doing a type check in this to see if a given interface-satisfying value _is_ already an `api.Repository`? (genuine question, I'm not sure)